sourcekitten.xcodeproj/xcshareddata/xcschemes/sourcekitten.xcscheme

```diff
@@ -17,7 +17,7 @@
                BlueprintIdentifier = "D0E7B63119E9C64500EDBA4D"
                BuildableName = "sourcekitten.app"
                BlueprintName = "sourcekitten"
-               ReferencedContainer = "container:sourcekitten.xcodeproj">
+               ReferencedContainer = "container:SourceKitten.xcodeproj">
```

Comment:
It seems `sourcekitten.xcodeproj` are mixed with `SourceKitten.xcodeproj`.
Directory name is `sourcekitten.xcodeproj`, but `sourcekitten.xcodeproj/project.pbxproj` thinks itself as `SourceKitten.xcodeproj`.